vendor/ember-cli-qunit/test-container-styles.css

```diff
@@ -11,5 +11,8 @@
   margin: 0 auto;
 }
 #ember-testing {
-  zoom: 50%;
+  width: 200%;
+  height: 100%;
```

Comment:
I don't think we want a specified `height`. Current behavior is that `#ember-testing` should grow as the content does so that way we still get scroll bars and such.